api/src/server/event.rs

```diff
@@ -34,9 +34,13 @@ where
         car_blocks.insert(cid, bytes);
     }
     let event_bytes = get_block(&event_cid, &car_blocks, store).await?;
-    let event: unvalidated::Event =
+    let event: unvalidated::Event<Ipld> =
         serde_ipld_dagcbor::from_slice(&event_bytes).context("decoding event")?;
-    let init_id = match event {
+    let (init_id, payload) = match event {
```

Comment:
nitpick, can we call this `init_payload` so it's clear it's not the payload of the event itself?